files/scripts/core_cleanup.py

```diff
@@ -1,15 +1,31 @@
 #!/usr/bin/env python3
 
 import os
+import re
 from collections import defaultdict
-from datetime import datetime
+from datetime import datetime, date, timedelta
 
 from sonic_py_common.logger import Logger
 
 SYSLOG_IDENTIFIER = 'core_cleanup.py'
 CORE_FILE_DIR = '/var/core/'
+KERNEL_DUMP_DIR = '/var/dump/'
 MAX_CORE_FILES = 4
+EXPIRE_DAYS = 90
 
+def delete_file(file_path):
+    try:
+        os.remove(file_path)
+    except:
+        logger.log_error('Unexpected error occured trying to delete {}'.format(file_path))
```

Comment:
>log_error [](http://example.com/codeflow?start=15&length=9)

Print exception object, because it is swallowed. Should we use more specific exception type?

Reply:
There are multiple exceptions can throw by os.remove